src/main/java/mate/academy/springboot/util/SortUtil.java

```diff
@@ -0,0 +1,31 @@
+package mate.academy.springboot.util;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SortUtil {
+    public Sort sort(String sortBy) {
+        List<Sort.Order> orders = new ArrayList<>();
+        if (sortBy.contains(":")) {
+            String[] sortingFields = sortBy.split(";");
+            for (String field: sortingFields) {
+                Sort.Order order;
+                if (field.contains(":")) {
+                    String[] fieldsAndDirections = field.split(":");
+                    order = new Sort.Order(Sort.Direction.valueOf(fieldsAndDirections[1]),
+                            fieldsAndDirections[0]);

```

Comment:
Replace magic values with constants. Do not forget first homeworks)

Reply:
I got it))